docs/adding_packages/build_and_package.md

```diff
@@ -6,72 +6,13 @@ Both methods often use build helpers to build binaries and install them into the
 <!-- toc -->
 ## Contents
 
-  * [Build Method](#build-method)
-  * [Package Method](#package-method)
   * [Build System Examples](#build-system-examples)
     * [Header Only](#header-only)
     * [CMake](#cmake)
     * [Autotools](#autotools)
     * [No Upstream Build Scripts](#no-upstream-build-scripts)
     * [System Packages](#system-packages)<!-- endToc -->
 
-## Build Method
```

Comment:
Build is well covered by Conan docs already:

- https://docs.conan.io/2/reference/conanfile/methods/build.html
- https://docs.conan.io/2/tutorial/creating_packages/build_packages.html

Reply:
Maybe link to Conan 2 docs directly?

Reply:
Addressed by cf533b2e049c2c9b099a74fa702cebbd2dc3d94b